test/test_transforms.py

```diff
@@ -525,14 +525,80 @@ def test_observationnorm(
                     assert (observation_spec[key].space.minimum == loc).all()
                     assert (observation_spec[key].space.maximum == scale + loc).all()
 
-    @pytest.mark.parametrize("batch", [[], [1], [3, 2]])
-    @pytest.mark.parametrize(
-        "keys", [["next_observation", "some_other_key"], ["next_observation_pixels"]]
-    )
+    def test_catframes_transform_observation_spec(self):
+        N = 4
+        key1 = "first key"
+        key2 = "second key"
+        keys = [key1, key2]
+        cat_frames = CatFrames(N=N, keys=keys)
+        mins = [0, .5]
+        maxes = [.5, 1]
+        observation_spec = CompositeSpec(
+            **{
+                key: NdBoundedTensorSpec(space_min, space_max, (1, 32, 32), dtype=torch.double)
+                for key, space_min, space_max in zip(keys, mins, maxes)
+            }
+        )
+
+        result = cat_frames.transform_observation_spec(observation_spec)
+        observation_spec = CompositeSpec(
+            **{
+                key: NdBoundedTensorSpec(space_min, space_max, (1, 32, 32), dtype=torch.double)
+                for key, space_min, space_max in zip(keys, mins, maxes)
+            }
+        )
+
+        print()
```

Comment:
do we have to keep it?

Reply:
Wow my mistake from debugging. Sorry for that I will fix.